web/portal/brand/src/entities/Company/Form.tsx

```diff
@@ -104,7 +109,12 @@ const Form = (props: EntityFormProps): JSX.Element | null => {
     edit &&
       !isResidential && {
         legend: _('Platform data'),
-        fields: ['outgoingDdi', 'outgoingDdiRule'],
+        fields: [
+          !isWholesale && !isRetail && 'outgoingDdi',
+          !isWholesale && !isRetail && 'outgoingDdiRule',
+          'mediaRelaySets',
+          (isVpbx || isResidential) && 'distributeMethod',
```

Comment:
Friendly reminder. These fields are managed by platform administator, so they must not be visible in client portal. They could be also removed from API response if we want.

Reply:
:+1: